[0.021s] Invoking command in '/home/ghani/my_ws/build/customized_srv_msg': CMAKE_PREFIX_PATH=/opt/ros/galactic /usr/bin/cmake --build /home/ghani/my_ws/build/customized_srv_msg -- -j6 -l6
[0.093s] -- Found ament_cmake: 1.1.5 (/opt/ros/galactic/share/ament_cmake/cmake)
[0.127s] -- Found PythonInterp: /usr/bin/python3 (found suitable version "3.8.10", minimum required is "3") 
[0.127s] -- Using PYTHON_EXECUTABLE: /usr/bin/python3
[0.274s] -- Found rosidl_default_generators: 1.1.1 (/opt/ros/galactic/share/rosidl_default_generators/cmake)
[0.284s] -- Using all available rosidl_typesupport_c: rosidl_typesupport_fastrtps_c;rosidl_typesupport_introspection_c
[0.296s] -- Found rosidl_adapter: 2.2.1 (/opt/ros/galactic/share/rosidl_adapter/cmake)
[0.545s] -- Using all available rosidl_typesupport_cpp: rosidl_typesupport_fastrtps_cpp;rosidl_typesupport_introspection_cpp
[0.621s] -- Found PythonInterp: /usr/bin/python3 (found version "3.8.10") 
[0.851s] CMake Error at /opt/ros/galactic/share/rosidl_adapter/cmake/rosidl_adapt_interfaces.cmake:60 (message):
[0.851s]   execute_process(/usr/bin/python3 -m rosidl_adapter --package-name
[0.851s]   customized_srv_msg --arguments-file
[0.851s]   /home/ghani/my_ws/build/customized_srv_msg/rosidl_adapter__arguments__customized_srv_msg.json
[0.851s]   --output-dir
[0.851s]   /home/ghani/my_ws/build/customized_srv_msg/rosidl_adapter/customized_srv_msg
[0.851s]   --output-file
[0.851s]   /home/ghani/my_ws/build/customized_srv_msg/rosidl_adapter/customized_srv_msg.idls)
[0.851s]   returned error code 1:
[0.852s] 
[0.852s]   Error processing 'float32 x, y' of 'customized_srv_msg/Goal_Goal': ''x,' is
[0.852s]   an invalid field name.  It should have the pattern
[0.852s]   '^(?!.*__)(?!.*_$)[a-z][a-z0-9_]*$''
[0.852s] 
[0.852s]   Traceback (most recent call last):
[0.852s] 
[0.852s]     File "/usr/lib/python3.8/runpy.py", line 194, in _run_module_as_main
[0.853s]       return _run_code(code, main_globals, None,
[0.853s]     File "/usr/lib/python3.8/runpy.py", line 87, in _run_code
[0.853s]       exec(code, run_globals)
[0.853s]     File "/opt/ros/galactic/lib/python3.8/site-packages/rosidl_adapter/__main__.py", line 19, in <module>
[0.853s]       sys.exit(main())
[0.853s]     File "/opt/ros/galactic/lib/python3.8/site-packages/rosidl_adapter/main.py", line 53, in main
[0.853s]       abs_idl_file = convert_to_idl(
[0.854s]     File "/opt/ros/galactic/lib/python3.8/site-packages/rosidl_adapter/__init__.py", line 29, in convert_to_idl
[0.854s]       return convert_action_to_idl(
[0.854s]     File "/opt/ros/galactic/lib/python3.8/site-packages/rosidl_adapter/action/__init__.py", line 28, in convert_action_to_idl
[0.854s]       action = parse_action_string(package_name, input_file.stem, content)
[0.854s]     File "/opt/ros/galactic/lib/python3.8/site-packages/rosidl_adapter/parser.py", line 892, in parse_action_string
[0.854s]       goal_message = parse_message_string(
[0.854s]     File "/opt/ros/galactic/lib/python3.8/site-packages/rosidl_adapter/parser.py", line 515, in parse_message_string
[0.854s]       fields.append(Field(
[0.854s]     File "/opt/ros/galactic/lib/python3.8/site-packages/rosidl_adapter/parser.py", line 353, in __init__
[0.854s]       raise NameError(
[0.854s] 
[0.854s]   NameError: 'x,' is an invalid field name.  It should have the pattern
[0.854s]   '^(?!.*__)(?!.*_$)[a-z][a-z0-9_]*$'
[0.854s] 
[0.854s] Call Stack (most recent call first):
[0.854s]   /opt/ros/galactic/share/rosidl_cmake/cmake/rosidl_generate_interfaces.cmake:130 (rosidl_adapt_interfaces)
[0.854s]   CMakeLists.txt:12 (rosidl_generate_interfaces)
[0.854s] 
[0.854s] 
[0.854s] -- Configuring incomplete, errors occurred!
[0.855s] See also "/home/ghani/my_ws/build/customized_srv_msg/CMakeFiles/CMakeOutput.log".
[0.858s] make: *** [Makefile:1228: cmake_check_build_system] Error 1
[0.860s] Invoked command in '/home/ghani/my_ws/build/customized_srv_msg' returned '2': CMAKE_PREFIX_PATH=/opt/ros/galactic /usr/bin/cmake --build /home/ghani/my_ws/build/customized_srv_msg -- -j6 -l6
